cli/turbo.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../docs/public/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "build": {
+      "outputs": ["turbo", "turbo.exe"]
+    },
+
```

Comment:
```suggestion
```
This is definitely not going to do what we want. We should probably pop this out because it doesn't work like this.

Reply:
it _should_ do the same thing as it did before this change. are you saying it's already broken in main or because of this change?